dotCMS/src/main/java/com/liferay/portal/language/LanguageUtil.java

```diff
@@ -562,10 +563,23 @@ public static Locale getDefaultLocale(HttpServletRequest req){
 	 * @return
 	 */
     public static String getLiteralLocale(final String languageCode, final String countryCode) {
-        final String newCountryCode = (countryCode == null || countryCode.isEmpty()) ? Config
-                        .getStringProperty("language.default.country.mapping." + languageCode,
-                                        languageCode)
+        final String newCountryCode = (countryCode == null || countryCode.isEmpty()) ? getDefaultCountry(languageCode)
                         : countryCode;
-        return languageCode + "_" + newCountryCode;
+        return new StringBuilder(languageCode).append("_").append(newCountryCode).toString();
+    }
+
+    public static String getDefaultCountry(final String languageCode) {
+        final String defaultCountryCode = Config.getStringProperty("language.default.country.mapping." + languageCode,languageCode);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "defaultCountryCode". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)